### ISSUE TITLE
Error when using vite in production

### ISSUE THREAD

**Issue by craiggeil** [digitalfortress-tech/typeahead-standalone#63]
### What happened?

Using vite v4.3.7 with vue 3.3.4. Latest version of typeahead-standalone. 

Inserted the colors example into my page. Works great in dev mode but as soon as I build using vite and run in production I get this error --

Uncaught TypeError: Cannot read properties of undefined (reading 'webpackChunktypeahead')
    at index-e65366ba.js:2156:108666
    at index-e65366ba.js:2156:108754
    at index-e65366ba.js:2156:108817
    at index-e65366ba.js:2156:96746
    at index-e65366ba.js:2156:96751
    at index-e65366ba.js:2156:108822

### Affected version

Latest

### Reproduction URL

none

### How to reproduce?

Use standard vue 3 example. Insert example code from typeahead-standalone. Run vite build. Deploy to web site. 

### Browsers

Chrome

### Relevant log output

```Shell
Uncaught TypeError: Cannot read properties of undefined (reading 'webpackChunktypeahead')
    at index-e65366ba.js:2156:108666
    at index-e65366ba.js:2156:108754
    at index-e65366ba.js:2156:108817
    at index-e65366ba.js:2156:96746
    at index-e65366ba.js:2156:96751
    at index-e65366ba.js:2156:108822
```

**Comment by niketpathak** [comment]
Hello, thank you for reporting this. Will look into it asap.

**Comment by samturrell** [comment]
Any updates on this? Having the same issue 😬

**Comment by samturrell** [comment]
Looks to be an issue with `globalObject: 'this',` in vite environments. Setting to `global` or `globalThis` seems to resolve it, but unsure how this will effect other environments

**Comment by craiggeil** [comment]
Any updates on this?

**Comment by niketpathak** [comment]
The migration is in progress. Will update as soon as done.

**Comment by wtfz** [comment]
> Looks to be an issue with `globalObject: 'this',` in vite environments. Setting to `global` or `globalThis` seems to resolve it, but unsure how this will effect other environments

@samturrell Did you happen to make it work in production build with this? I'm still new with vite, not sure how to set this in `vite.config.js` like you suggest.

**Comment by craiggeil** [comment]
Ended up removing this component. Looks like the author doesn't have time to work on this anymore. Prime has an excellent auotcomplete component that works way better than this

**Comment by samturrell** [comment]
> > Looks to be an issue with `globalObject: 'this',` in vite environments. Setting to `global` or `globalThis` seems to resolve it, but unsure how this will effect other environments
> 
> @samturrell Did you happen to make it work in production build with this? I'm still new with vite, not sure how to set this in `vite.config.js` like you suggest.

Hi @wtfz - I forked the repo and did a fix on this package: https://www.npmjs.com/package/@netsells/typeahead-standalone

The fix worked for my usecase, but maybe not for yours, and i'm unfortunately not able to provide any support

**Comment by niketpathak** [comment]
For future readers, this issue has been fixed in [v4.25.0](https://github.com/digitalfortress-tech/typeahead-standalone/releases/tag/v4.25.0)
